examples/export.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import datafusion
+from datafusion import col
+import pyarrow
+import polars as pl
+
+
+# create a context
+ctx = datafusion.SessionContext()
+
+# create a RecordBatch and a new datafusion DataFrame from it
+batch = pyarrow.RecordBatch.from_arrays(
+    [pyarrow.array([1, 2, 3]), pyarrow.array([4, 5, 6])],
+    names=["a", "b"],
+)
+df = ctx.create_dataframe([[batch]])
+# Dataframe:
+# +---+---+
+# | a | b |
+# +---+---+
+# | 1 | 4 |
+# | 2 | 5 |
+# | 3 | 6 |
+# +---+---+
+
+# export to pandas dataframe
+pandas_df = df.to_pandas()
+assert pandas_df.shape == (3, 2)
+
+# export to PyArrow table
+arrow_table = df.to_arrow_table()
+assert arrow_table.shape == (3, 2)
+
+# export to Polars dataframe
+polars_df = pl.DataFrame(df.to_arrow_table())
+assert polars_df.shape == (3, 2)
```

Comment:
Do you think it's worthwhile introducing a `to_polars()` helper function?

Reply:
Yes, I think that would be great

Reply:
Thanks for the feedback - added `to_polars()` method & unit tests for it